### PR TITLE
Watch progress of verification checks

### PR DESCRIPTION
**Pull request by ftreesmilo**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[x ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
emit 2 new events on torrent so that you can see when a piece is verified or invalidated when verifying

**Which issue (if any) does this pull request address?**
n/a

**Is there anything you'd like reviewers to focus on?**
n/a

### CODE REVIEW

**Comment by ftreesmilo** [comment]
npm run test-node passes locally for me, not sure why the appveyor failed.

**Comment by ftreesmilo** [comment]
please review this as well

File: lib/torrent.js [hunk 1]
```diff
@@ -1545,10 +1547,7 @@ class Torrent extends EventEmitter {
         if (hash === self._hashes[index]) {
           if (!self.pieces[index]) return
           self._debug('piece verified %s', index)
-
-          self.pieces[index] = null
-          self._reservations[index] = null
-          self.bitfield.set(index, true)
+          self._markVerified(index)
```
Comment (dsernst) [inline lib/torrent.js:1550]: Why this code removed?

Comment (ftreesmilo) [reply lib/torrent.js:1550]: code reuse.  those same calls are in _markVerified

**Comment by ftreesmilo** [comment]
@feross can we please get this merged?

**Pull request closed without merging**